tests/kernel/build_git_kernel.pm

```diff
@@ -7,6 +7,8 @@
 # notice and this notice are preserved.  This file is offered as-is,
 # without any warranty.
 
+# Package: git-core ncurses-devel gcc flex bison libelf-devel libopenssl-devel
+# make dracut
```

Comment:
Is it planned to use multiple lines? I am thinking about human-readable vs machine-readable

Reply:
@martinsmarcelo said the parser script is able to handle this.